tests/integration/security/util/reachability/context.go

```diff
@@ -155,16 +157,15 @@ func (rc *Context) Run(testCases []TestCase) {
 				// TODO(https://github.com/istio/istio/issues/20460) We shouldn't need a retry loop
 				return rc.ctx.Config().ApplyYAML(c.Namespace.Name(), policyYAML)
 			})
+			ctx.Logf("[%s] [%v] Wait for config propagate to endpoints...", testName, time.Now())
+			if err := ik.WaitForConfigs(c.Namespace.Name(), policyYAML); err != nil {
```

Comment:
If I understand correctly, there's no ordering the apply is happening before Wait. and wait is robust to such sequencing?